docker/api/container.py

```diff
@@ -305,7 +305,7 @@ def start(self, container, binds=None, port_bindings=None, lxc_conf=None,
               dns=None, dns_search=None, volumes_from=None, network_mode=None,
               restart_policy=None, cap_add=None, cap_drop=None, devices=None,
               extra_hosts=None, read_only=None, pid_mode=None, ipc_mode=None,
-              security_opt=None, ulimits=None):
+              security_opt=None, ulimits=None, user_namespace=None):
```

Comment:
We are not adding new parameters to `start()` - anything but the container ID is deprecated anyways.